### PR TITLE
Add Waitress support for production

### PR DESCRIPTION
**Pull request by sbrunner**
With Gunicorn, we get one main process and one worker process. When the worker is killed the Pod steel alive!

Add resource metrics
Add psutil memory info metrics

### CODE REVIEW

**Comment by ger-benjamin** [comment]
Can you please add a ticket reference for our TS and understanding ?

**Comment by sbrunner** [comment]
@ger-benjamin you can timesheet in https://camptocamp.atlassian.net/browse/GSGMF-1747 :-)

**Comment by arnaud-morvan** [comment]
Not sure to understand why we need to duplicate the app, I'm used to switch from gunicorn to waitress just by changing the docker container command.

**Comment by sbrunner** [comment]
@arnaud-morvan I updated the README to clarify this point :-)

File: README.md [hunk 1]
```diff
@@ -608,6 +608,64 @@ def hello_get(request):
     return {'hello': True}
 ```
 
+## Waitress
+
+In production mode we usually use Gunicorn but we can also use Waitress.
+
+The advantage to use Waitress it that he creates only one process, that makes it easier to manage especially on Kubernetes:
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply README.md:615]: > Note that is also possible to create only one process with gunicorn.
When I try it, it's not the case

We get on worker process, and one main process, then when the worker is killed the Pod steel alive!

File: acceptance_tests/tests/docker-compose.yaml [hunk 2]
```diff
@@ -46,6 +43,7 @@ services:
 
   app2:
     <<: *app
+    image: camptocamp/c2cwsgiutils_test_app_waitress
     # Same as app but with 2 workers (and different Redis DB 2, broadcast_prefix, ports, and JSON log format)
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply acceptance_tests/tests/docker-compose.yaml:47]: Oups, updated :-)

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (arnaud-morvan) [reply README.md]: Ok thanks, I just copied this in PR description.

**Review by arnaud-morvan — APPROVED** [review]

**Comment by sbrunner** [comment]
Thanks @arnaud-morvan :-)

**Pull request merged**